.github/workflows/clang-format-checker.yml

```diff
@@ -17,22 +17,22 @@ jobs:
       - uses: actions/checkout@v3
         with:
           fetch-depth: 0
-      - name: install clang-format
-        run: sudo apt install -y clang-format
       - name: Comparing PR against target branch
         env:
           PR_NUMBER: ${{ github.event.number }}
           GH_TOKEN: ${{ github.token }}
         run: |
-            echo Comparing $GITHUB_BASE_REF vs HEAD
-            git diff -U0 --no-color origin/$GITHUB_BASE_REF..HEAD | clang-format-diff -p1 | tee format.diff
-            if [ -s "format.diff" ]
+            echo Comparing $PR_NUMBER vs target branch
+            git fetch origin refs/pull/$PR_NUMBER/head:pull/$PR_NUMBER
```

Comment:
Is this here by mistake?
Not like related to StringSwitch.

Reply:
I know where this belongs now:)
Not have enough knowledge to review it though:(

Reply:
This was a draft PR that I've been using to test the clang-format checker action.